### PR TITLE
working on three dialog translation

### LINKED ISSUE #629: [Feature]: Apply translation/localization to all text in the `ThreeDialog`
### Feature Summary

We should use `i18n` to translate the text in the `ThreeDialog`

### Detailed Description

We should pull all of the text in the ThreeDialog into the translation json files e.g. src/translations/en.json, src/translations/es.json, and src/translations/fr.json in under a key called "game.dialogs.threeDialog"

Look at src/views/LoginView for an example of a page that is correctly using $i18n to localize/translate content. The user-visible strings have been removed from the code and placed into a "login" section of the translation files in src/translations.

The page then imports and consumes the useI18n() composable to grab the t() function used to render/translate text by pulling from the appropriate language's json file. This is done using a setup() block of the page like so:

```
setup() {
const { t } = useI18n();
return { t };
},
```
From there, we can grab a given string by name in the appropriate locale using {{ t('login.joinDiscord') }} since each translation json in this case has a login attribute for all the text in the login page, with a joinDiscord child attribute for the text to show in the 'Join our discord community' button.

We should do the same thing to the ThreeDialog by extracting all the text in the page into a rules section of each translation json file (e.g. src/translations/en.json) and then configuring ThreeDialog.vue with the useI18n() composable and rendering all its text with t() to translate it to the user's selected locale.

All the text for the ThreeDialog should be stored in each language file under the key "game.dialogs.threeDialog" (generally we have one top-level key for each page e.g. "game" and "login", then specific sub-keys for components in that page (additionally grouping all the dialogs under the sub-key "game.dialogs"

The end result of this change should be the ability to see the text of the ThreeDialog in your preferred locale.

### PR DESCRIPTION
**Pull request by AlexVCS**
## Issue number
#629 

- Working to Resolve #629 

## Please check the following

- [✅ ] Do the tests still pass? (see [Run the Tests](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#run-the-tests))
- [ ] Is the code formatted properly? (see [Linting (Formatting)](https://github.com/cuttle-cards/cuttle/blob/main/docs/setup-development.md#linting-formatting))
- For New Features:
  - [ ] Have tests been added to cover any new features or fixes?
  - [ ] Has the documentation been updated accordingly?

## Please describe additional details for testing this change
I want to look at what I've added but am not sure where to access it. I'll have to find out where it lives and check that it works as intended.

### CODE REVIEW

File: src/translations/es.json [hunk 1]
```diff
@@ -56,6 +56,10 @@
           "chronologically": "Cronológicamente",
           "rank": "Por Rango"
         }
+      },
+      "threeDialog": {
+        "title": "Seleccione una tarjeta de Scrap",
+        "resolve": "Resolver"
```
[the remark this answers is not in the document]
Comment (AlexVCS) [reply src/translations/es.json:62]: any thoughts @itsalaidbacklife?

[the remark this answers is not in the document]
Comment (Haviles04) [reply src/translations/es.json:62]: It looks like someone beat us to it, it's already like this in the main repo. I think if you update your branch to match main you'll probably get a merge conflict that resolves this. no pun intended lol

[the remark this answers is not in the document]
Comment (AlexVCS) [reply src/translations/es.json]: sorry for the delay, haha thank you!

**Comment by itsalaidbacklife** [comment]
> Any feedback on this? I'm not sure why the e2e install isn't working correctly

It looks like cypress is failing to parse the json in the `en.json` file, likely due to a syntax error

File: src/translations/en.json [hunk 2]
```diff
@@ -64,16 +64,8 @@
...
-        "oneOff": "as a one-off",
-        "target": "targeting ",
-        "counter": "to Counter",
-        "cannotCounter": "You cannot Counter, because ",
-        "reasons": {
-          "opponentWithQueen": "your opponent has a queen",
-          "noTwoPresent": "you do not have a two"
-        }
+      "threeDialog": {
+        "title": "Select a Card from Scrap",
```
Comment (itsalaidbacklife) [inline src/translations/en.json:68]:
```suggestion
        "title": "Select a Card from Scrap"
```
I think this is the issue. The trailing comma on the last line of a block is invalid json

File: src/translations/es.json [hunk 3]
```diff
@@ -59,21 +59,8 @@
...
-        "oneOff": "como un one-off",
-        "target": "apuntar ",
-        "counter": "para contrarrestar",
-        "cannotCounter": "No puedes contrarrestar, porque",
-        "reasons": {
-          "opponentWithQueen": "Tu oponente tiene una queen",
-          "noTwoPresent": "no tienes un dos"
-        }
+      "threeDialog": {
+        "title": "Seleccione una tarjeta de Scrap",
```
Comment (itsalaidbacklife) [inline src/translations/es.json:63]:
```suggestion
        "title": "Seleccione una tarjeta de Scrap"
```

**Comment by AlexVCS** [comment]
Do I need to rebase? I keep getting this when I try to push the right changes and I pulled in the latest changes I think:

 ! [rejected]        ThreeDialog-translation -> ThreeDialog-translation (non-fast-forward)
error: failed to push some refs to 'https://github.com/AlexVCS/cuttle.git'
hint: Updates were rejected because the tip of your current branch is behind
hint: its remote counterpart. Integrate the remote changes (e.g.
hint: 'git pull ...') before pushing again.
hint: See the 'Note about fast-forwards' in 'git push --help' for details.

**Comment by AlexVCS** [comment]
My branches say up to date when I do 
`git pull https://github.com/cuttle-cards/cuttle.git main`

**Comment by itsalaidbacklife** [comment]
@AlexVCS that indicates that you need to `git pull`. It's possible that your previous pull was rejected due to local changes that aren't pushed yet. So you could try resolving this with `git pull --rebase` before pushing.

If you're unable to pull with a rebase, we can hash out the details of the git error in discord

**Review by itsalaidbacklife — APPROVED** [review]
Nice work. Thanks for your contribution!

**Pull request merged**